.github/workflows/label.yml

```diff
@@ -2,14 +2,20 @@
 # paths that are modified in the pull request.
 
 name: 🔖 Labeler
-on: [pull_request_target]
+on:
+  pull_request_target:
+    paths:
+      - '**'
```

Comment:
what's the difference to how it was before?

Reply:
it looks more complicated to me now

Reply:
> what's the difference to how it was before?

Doesn't really make a difference